clarity_ext/service/dilution/service.py

```diff
@@ -422,8 +422,9 @@ def updated_source_vol(self):
 
     @property
     def update_info(self):
+        end_volume = self.pipette_sample_volume + self.pipette_buffer_volume
```

Comment:
This looks like it's the same as the `pipette_total_volume` property, can you use that instead?